OMEdit/OMEditLIB/Util/Utilities.h

```diff
@@ -417,6 +417,8 @@ class QDetachableProcess : public QProcess
   Q_OBJECT
 public:
   QDetachableProcess(QObject *pParent = 0);
+
+  using QProcess::start;
```

Comment:
Is this necessary?

Reply:
yes, else we cannot call the start(QString) method overload

Reply:
`using` pulls in all the overloads.

Reply:
@adeas31 good point